CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   More info about helmet configuration https://helmetjs.github.io/docs/
 - Add config `users.tokenInHeader` which allows to send token in header instead in query. Require to set on true same config in vsf-api.
 - Make calculation of bundled products price by options optional - @cewald (#4556)
+- Reduce initial client-side bundle-size by lazy-loading `i18n` translations - @cewald (#4821)
```

Comment:
can you move it to ?
```
## [1.13.0] - UNRELEASED

### Added
- Reduce initial client-side bundle-size by lazy-loading `i18n` translations - @cewald (#4821)
```

Reply:
Latest was on 1.12.2 – I added 1.13.0 or is there another integration branch?

Reply:
its perfect 👌 thank you